lib/ops/opsservice/configure.go

```diff
@@ -673,6 +679,10 @@ func (s *site) getPlanetMasterSecretsPackage(ctx *operationContext, p planetMast
 						s.domainName, host}, "."))
 				}
 			}
+
+			// Don't rotate the APIServer key, the secret is currently used for validating serviceaccounttokens
+			// TODO(security) enable rotation of secret for apiserver/serviceaccounttokens
+			privateKeyPEM = apiserverKeyPair.KeyPEM
```

Comment:
I know this way the changes are kept to a minimum, but I'd want the apiserver case handled visually separately:
```go
if name == constants.APIServerKeyPair {
	keyPair, err := authority.GenerateCertificate(req, caKeyPair, apiserverKeyPair.KeyPEM, defaults.CertificateExpiry)
	if err != nil {
		return nil, trace.Wrap(err)
	}
} else {
	keyPair, err := authority.GenerateCertificate(req, caKeyPair, nil, defaults.CertificateExpiry)
	if err != nil {
		return nil, trace.Wrap(err)
	}
}
```